components/org.wso2.carbon.identity.api.dispatcher/src/main/webapp/WEB-INF/beans.xml

```diff
@@ -48,6 +48,10 @@
     <import resource="classpath:META-INF/cxf/cors-server-v1-cxf.xml"/>
     <import resource="classpath:META-INF/cxf/user-functionality-v1-cxf.xml"/>
     <import resource="classpath:META-INF/cxf/rfc-server-v1-cxf.xml"/>
+
+    <!-- Legacy identity APIs   -->
+    <import resource="classpath:META-INF/cxf/config-mgt-server-v1-cxf.xml"/>
```

Comment:
Do we have an import related to entitlement api?

Reply:
The entitlement CXF servlet doesn't have interface and implementation separately.  
https://github.com/wso2/carbon-identity-framework/pull/3442/files#diff-1b5ecd7afff40530658f5b9e8933a62e5daeedda7b606193bad22c15a80bbc7dL28